engine/src/main/java/io/camunda/zeebe/engine/api/ProcessingScheduleService.java

```diff
@@ -28,4 +30,6 @@ default void runAtFixedRate(final Duration delay, final Runnable task) {
           }
         });
   }
+
+  void runAtFixedRate(final Duration delay, final Task task);
```

Comment:
:thinking: Didn't we removed that on purpose why we have that again?

Reply:
see other comment on quality level of this class